src/dotnet/APIView/APIView/APIView.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
```

Comment:
Any reason for disabling attributes?

Reply:
I was having issues with building the solution due to duplicate attributes. Is there something better I'd be able to do to fix that?

Reply:
git clean should help, or just removing `bin` and `obj` folders from your project directory.